mono/mini/mini.c

```diff
@@ -3685,6 +3685,8 @@ mini_method_compile (MonoMethod *method, guint32 opts, MonoDomain *domain, JitFl
 		mono_local_deadce (cfg);
 	if (cfg->opt & MONO_OPT_ALIAS_ANALYSIS)
 		mono_local_alias_analysis (cfg);
+	if (cfg->opt & MONO_OPT_DIVREM && cfg->has_divisions)
+		mono_local_div_imm_removal (cfg);
```

Comment:
a whole pass just for replacing potential `div`s? hum.  don't we have infrastructure to add such peephole optimization?  is there any reason why it's after `deadce` and `alias analysis`?